src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -22,7 +22,7 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends"), getAppointmentSet("23:59 SUN")),
+                getTagSet("friends"), getAppointmentSet("23:59 SUN", "00:00 MON")),
```

Comment:
In the future, it is better to make this change in a separate commit! It just helps to organise the changes together.